compiler-plugin/src/main/java/module-info.java

```diff
@@ -21,4 +21,8 @@
     requires io.ballerina.tools.api;
     requires io.ballerina.parser;
     requires io.ballerina.stdlib.graphql.commons;
+
+    exports io.ballerina.stdlib.graphql.compiler;
+//    exports io.ballerina.stdlib.graphql.compiler.schema.generator;
+//    exports io.ballerina.stdlib.graphql.compiler.service;
```

Comment:
Shall we remove the commented-out lines?

Reply:
fixed with [e67b64d](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1238/commits/e67b64dd275c83e33581684174c4aafd9843a20e)